Sources/Commands/SwiftTool.swift

```diff
@@ -366,6 +357,19 @@ public class SwiftTool<Options: ToolOptions> {
         }
     }
 
+    func buildPlan(graph: PackageGraph, config: Build.Configuration) throws -> BuildPlan {
```

Comment:
I think its worth moving the config option down to `SwiftTool` because `swift run` is going to need to too.